source/_posts/2022-12-20-year-of-voice.markdown

```
@@ -23,6 +23,10 @@ _**TL;DR**: It is our goal for 2023 to let users control Home Assistant in their
  - [Year of the Voice - Chapter 4](/blog/2023/10/12/year-of-the-voice-chapter-4-wakewords/) (October 12, 2023)
  - [Year of the Voice - Chapter 5](/blog/2023/12/13/year-of-the-voice-chapter-5/) (December 13, 2023)
  - [Voice - Chapter 6](/blog/2024/02/21/voice-chapter-6/) (February 21, 2024)
+ - [Voice - Chapter 7](/blog/2024/06/26/voice-chapter-7/) (June 26, 2024)
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 0; Actual: 1

(MD007, ul-indent)